components/brave_rewards/browser/rewards_notification_service_impl.cc

```diff
@@ -23,9 +23,9 @@
 namespace brave_rewards {
 
 RewardsNotificationServiceImpl::RewardsNotificationServiceImpl(Profile* profile)
-    : profile_(profile),
+    : profile_(profile)
 #if BUILDFLAG(ENABLE_EXTENSIONS)
-      extension_rewards_notification_service_observer_(
+      , extension_rewards_notification_service_observer_(
```

Comment:
this is likely going to fail lint checks. Can we just move initialization into the constructor body?